source/rust_verify/src/main.rs

```diff
@@ -95,13 +95,8 @@ pub fn main() {
         );
 
     if is_direct_rustc_call {
-        args.insert(0, program);
-        match rust_verify::driver::run_rustc_compiler_directly(&args) {
-            Ok(()) => return,
-            Err(_) => {
-                std::process::exit(1);
-            }
-        }
+        args.insert(0, program.clone());
+        rust_verify::driver::run_rustc_compiler_directly(&args);
```

Comment:
> Can you try cargo verus in this branch (source/examples/cargo-verus/run.sh) before merging? I got an error Argument to option 'V' missing.

I think the cause is that this code used to always exit here, but now continues after `run_rustc_compiler_directly`, leading to a later spurious error.

Reply:
From what I could tell, `RunCompiler::run` now panics if there are errors (by eventually calling [`FatalError::raise`](https://github.com/rust-lang/rust/blob/master/compiler/rustc_span/src/fatal_error.rs#L14), so I thought this would also abort here.

Reply:
Oh, you're right, missed the fact that this returned on a successful run too.

Reply:
Fixed in latest version of the PR.